packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingMarks/DrawingMarks.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import DrawingToolRoot from '@plugins/tasks/DrawingTask/components/tools/DrawingToolRoot'
+import DeleteButton from '@plugins/tasks/DrawingTask/components/tools/DeleteButton'
+
+function DrawingMarks ({ activeMark, onDelete, onDeselectMark, onPointerDown, onPointerUp, onSelectMark, svg, tool }) {
```

Comment:
I don't think `onPointerDown` and `onPointerUp` are used any more.

Reply:
`svg` is being used as a prop in so many places that it would make sense to move it into context.

Reply:
Yep, `onPointerDown` and `onPointerUp` aren't being used.